modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc

```diff
@@ -5,19 +5,19 @@
 // Retains the context of the parent assembly if this assembly is nested within another assembly.
 // For more information about nesting assemblies, see: https://redhat-documentation.github.io/modular-docs/#nesting-assemblies
 // See also the complementary step on the last line of this file.
-ifdef::context[:parent-context: {context}]
+ifdef::context[:parent-context-of-${module_id}: {context}]
 
 // Base the file name and the ID on the assembly title. For example:
 // * file name: my-assembly-a.adoc
 // * ID: [id='my-assembly-a']
 // * Title: = My assembly A
 
 // The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
-[id='a-collection-of-modules']
-// If the assembly is reused in other assemblies in a guide, include {context} in the ID: [id='a-collection-of-modules-{context}'].
-= A collection of modules
+[id='${module_id}']
```

Comment:
I wasn't sure here: Do we want to default to assembly IDs having context or not? Alternatively, should I add an `ifdef` statement to take care of the two scenarios automatically?